src/utils/utils.go

```diff
@@ -174,3 +177,27 @@ func GetHostIpsFromInventory(inventory *models.Inventory) ([]string, error) {
 	}
 	return ips, nil
 }
+
+func AddStringToTarGz(w io.Writer, data string, fileName string) error {
```

Comment:
`WriteCompressedFileToPipe`

Reply:
it is not pipe. it is any writer. i like this name.

Reply:
it describes what it does. writes string to tar gz. can change add to write